README.md

```diff
@@ -48,17 +56,19 @@ let pinecone: PineconeClient = pinecone_sdk::pinecone::default_client().expect("
 ## Create Index
 
 ### Create serverless index
+
 The following example creates a serverless index in the `us-east-1` region of AWS. For more information on serverless and regional availability, see [Understanding indexes](https://docs.pinecone.io/guides/indexes/understanding-indexes#serverless-indexes)
+
 ```rust
 use pinecone_sdk::pinecone::PineconeClientConfig;
-use pinecone_sdk::pinecone::control::{Metric, Cloud, WaitPolicy, IndexModel};
+use pinecone_sdk::models::{Metric, Cloud, WaitPolicy, IndexModel};
```

Comment:
I think most of the code examples in here need this update. I can't remember if the types were originally exported from the modules and that was changed, but I needed to write things differently when I was testing locally.